api/controller/blog.js

```diff
@@ -9,4 +10,61 @@ exports.getBlog = function(req,res) {
 		
 		res.json(blog);
 	});
-};
\ No newline at end of file
+};
+
+exports.likeBlogComment = function(req, res, next) {
+    Blog.findOne({
+        _id: req.params.blogId,
+        comments: req.params.commentId
+    }, function(err, blog) {
+        if (err) {
+            next(err);
+            return;
+        }
+        else if (!blog) {
+            res.status(403).json({
+                error: 'Blog does not exist.'
+            });
+            return;
+        }
+
+        Comment.findById(req.params.commentId, function(err, comment) {
```

Comment:
I dont think we are storing commentd in the mongo db. This comment schema will be stored as a value in a particular blog . Correct me If I am wrong.